Deprecated/stop_all_C7.py

```diff
@@ -9,7 +9,7 @@
 
 from __future__ import absolute_import, division, print_function
 
-from hera_mc import mc, cm_utils, part_connect, cm_handling, geo_handling, cm_hookup, cm_part_revisions
+from hera_mc import mc, cm_utils, part_connect, cm_handling, cm_hookup, cm_part_revisions
```

Comment:
Might make sense not to touch the scripts in `Deprecated/` once they've served their purpose? Obviously not a big deal though.